### ISSUE TITLE
Add ability to easily switch between TaskWarrior databases

### ISSUE THREAD

**Issue by thehunmonkgroup** [vit-project/vit#235]
Feature request culled from #233 

An approach like this would allow pretty easy switching between TaskWarrior databases:
 * A configuration option to specify 'shortcuts' to various task databases
 * A new action that would allow a user to enter a shortcut name, which would then reload VIT with the task database the shortcut refers to.

Should be pretty efficient, for the following reasons:

 * The action would be bound to a key, just hit the keyboard shortcut and enter the alias. Both history and tab completion would be supported for alias name
 * I believe you could also map additional keyboard shortcuts to do a direct switch to a particular task database.

**Comment by fdela** [comment]
Only somewhat related, but it would be nice if vit could accept a command line parameter specifying data folder to use.
That could be used to feed _task_ with e.g. a default rc.data.location override, or possibly setting the TASKDATA envvar appropriately beforehand.
Alternatively allow passthrough command line param, which could e.g. also be used to specify alternate taskrc location (but there could be dragons)

Anyway, thanks for the good work!

**Comment by scottkosty** [comment]
I haven't tried it, but do things work as you want if you export TASKDATA to the location you want before calling vit?

**Comment by fdela** [comment]
Yes, that variable is used by TaskWarrior (and indirectly by vit) once it's set/exported, and things work as expected

You can use task rc.data.location:_alternateDataLocationDirectory_ rc:_alternateTaskrcConfFile_ _..._ or set associated TASKDATA and TASKRC environment variables.

See "CONFIGURATION FILE AND OVERRIDE OPTIONS" section in task(1) manpage for details.

**Comment by thehunmonkgroup** [comment]
@fdela I think your idea should probably be filed as a separate feature request.
